nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -155,32 +156,50 @@ def get_statistic_collector(
         quantizer_config: QuantizerConfig,
         inplace: bool,
         num_samples: int = None,
-    ) -> Union[PTMinMaxStatisticCollector, PTMeanMinMaxStatisticCollector]:
-        if (
-            range_estimator_params.min.statistics_type == StatisticsType.MIN
-            and range_estimator_params.min.aggregator_type == AggregatorType.MIN
-            and range_estimator_params.max.statistics_type == StatisticsType.MAX
-            and range_estimator_params.max.aggregator_type == AggregatorType.MAX
-        ):
-            collector_name = "min_max"
-
-        elif (
-            range_estimator_params.min.statistics_type == StatisticsType.MIN
-            and range_estimator_params.min.aggregator_type == AggregatorType.MEAN
-            and range_estimator_params.max.statistics_type == StatisticsType.MAX
-            and range_estimator_params.max.aggregator_type == AggregatorType.MEAN
+    ) -> TensorCollector:
+        collector_params = PTMinMaxAlgoBackend._default_collector_params(nncf_graph, target_point, quantizer_config)
+        collector_kwargs = collector_params.convert_statistic_params(per_sample_stats=False)
+
+        collector = TensorCollector(PTMinMaxTensorStatistic)
+        for params, container_key in zip(
+            [range_estimator_params.min, range_estimator_params.max],
+            [PTMinMaxTensorStatistic.MIN_STAT, PTMinMaxTensorStatistic.MAX_STAT],
         ):
-            collector_name = "mean_min_max"
-
-        else:
-            raise RuntimeError(
-                "The following range estimator parameters are not supported by PyTorch backend by now: "
-                f"{str(range_estimator_params)}"
-            )
-
-        return PTMinMaxAlgoBackend._statistic_collector_builder(
-            collector_name, nncf_graph, target_point, quantizer_config, num_samples
-        )
+            if not params.statistics_type in PT_REDUCERS_MAP:
+                raise RuntimeError(
+                    f"Statistic type: {params.statistics_type} is not supported for Torch PTQ backend yet."
+                )
+
+            if not params.aggregator_type in AGGREGATORS_MAP:
+                raise RuntimeError(
+                    f"Aggregator type: {params.aggregator_type} is not supported for Torch PTQ backend yet."
+                )
+
+            kwargs = {
+                "reduction_axes": collector_kwargs["reducers_axes"],
```

Comment:
Why `reducers_axes` instead of `reduction_axes`? What is the difference?

Reply:
This is misspelling, I'll fix

Reply:
Done